apps/website/src/app/docs/report/page.tsx

```diff
@@ -0,0 +1,232 @@
+"use client";
+
+import {
+  CheckResultCode,
+  ConsolidatedNameGuardReport,
+  Normalization,
+  Rating,
+} from "@namehash/nameguard";
+import { ReportBadge, ReportIcon } from "@namehash/nameguard-react";
+
+export default function RatingShieldsPage() {
+  return (
+    <div className="py-12 max-w-7xl mx-auto px-6 space-y-12">
+      <div className="divide-y divide-gray-100">
+        <h1 className="justify-center flex font-bold text-2xl my-8">
+          {"<"}ReportIcon {"/>"} documentation
+        </h1>
+
+        <div className="divide-y divide-gray-100">
+          <div className="grid grid-cols-6 gap-x-6 py-5 text-center font-medium">
+            <div></div>
+            <div>Success</div>
+            <div>Warn</div>
+            <div>Alert</div>
+            <div>Unknown</div>
+            <div>Loading</div>
+          </div>
+        </div>
+
+        <div className="divide-y divide-gray-100">
+          <div className="grid grid-cols-6 gap-x-6 py-5">
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportIcon />"}</pre>
+            </div>
+
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.pass)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.warn)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon data={getReportExampleData(Rating.alert)} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" hadLoadingError={true} />
+            </div>
+            <div className="flex items-center justify-center">
+              <ReportIcon name="notrab.eth" />
+            </div>
+          </div>
+          <div className="grid grid-cols-6 gap-x-6 py-5">
+            <div className="flex items-center font-mono">
+              <pre>{"<ReportIcon onClick />"}</pre>
```

Comment:
Could we make it more clear that this is a custom onClick override? My concern is that someone might think that the variation without a `onClick` passed should not be clickable. But that's not true. All cases are clickable, it's just that this case has an `onClickOverride`.